src/lib.rs

```diff
@@ -111,15 +118,15 @@ impl<T: Clone> Clone for Vc<T> {
     }
 }
 
-impl<T> Default for Vc<T> {
+impl<T, const R: usize> Default for RawVc<T, R> {
     /// Creates an empty `Vc<T>`.
     #[inline]
     fn default() -> Self {
         Self::new()
     }
 }
 
-impl<T> Vc<T> {
+impl<T, const S: usize> RawVc<T, S> {
```

Comment:
I'd prefer calling this `R` rather than `S`, just to keep the code consistent.

Reply:
That was originally my plan however later in the impl block R is already used for a Range type parameter (see https://github.com/jonhoo/atone/blob/79a65402b8dae6ff484b69042162078811df3bc2/src/lib.rs#L735C9-L735C9 for example)

Reply:
Ah, that's awkward! I think we should rename the `R` for the range type instead then. Make it `Range` — they don't all have to be single-letter names :)